std/examples/curl.ts

```diff
@@ -1,4 +1,4 @@
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
-const url = Deno.args[1];
-const res = await fetch(url);
+const url_ = Deno.args[1];
+const res = await fetch(url_);
```

Comment:
Why this change?

Reply:
Ah. This is currently an error

Reply:
Which is to say, with url it didn't compile. #2888 

This was one file I didn't actually verify (thought it was a sure thing).